docs/app/templates/docs/index.md

```diff
@@ -8,10 +8,66 @@ To install Ember Animated, run
 ember install ember-animated
 ```
 
+## Usage in Template Tag Format
```

Comment:
Would ❤️ to have documentation for template tag present. Whether with this improvement or not. It would greatly help folks to not to have to guess where to import from.